documentation/asciidoc/topics/proc_configuring_sites_automatically.adoc

```diff
@@ -4,6 +4,16 @@
 [role="_abstract"]
 Configure {ispn_operator} to establish cross-site views with {brandname} clusters.
 
+[NOTE]
+====
+Cross-site replication does not support the {k8s} CA service, so you must choose one of the following options:
+
+* Create a certificate and then add it to each client that connects to a {brandname} endpoint.
+* Use an online service to upload the certificate to your server.
+
+See link:https://access.redhat.com/solutions/6961656[Setting up Xsite locations url when using the nodeport/loadbalancer/routes in DG 8 OCP] (Red Hat KCS article)
```

Comment:
Our docs should be the canonical reference for configuring the Operator, so we shouldn't be outsourcing this information to KCS articles IMO.

Reply:
Agreed. It would only be possible for downstream content to which I would need to add an ifdef statement. WDYT?

Reply:
I don't think it's acceptable for downstream content either tbh, because the point remains that all required information should be contained with our docs. If the KCS was related to configuring an external component, or the platform, then I think it's acceptable, but in this case we're talking about the capabilities/limitations of our  actual product.